plugins/security/nexus-ldap-plugin/src/test/java/org/sonatype/security/realms/ldap/internal/ConnectionBlackListTest.java

```diff
@@ -29,19 +30,26 @@
 
 @Ignore("Test has not been run in a while, disabled in old surefire config")
```

Comment:
Test has been ignored since before https://issues.sonatype.org/browse/NEXUS-6591 (6 months) so I'm not sure what the point of updating code is without removing @Ignore and ensuring tests are functional